src/utils/ui/get_names.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function () {
```

Comment:
You should try to use native jquery syntax here, while this works, it's not ideal